http/obmc_shell.hpp

```diff
@@ -50,53 +50,27 @@ class Handler : public std::enable_shared_from_this<Handler>
             // ERROR
             if (session)
             {
-                session->close("Error creating ssh child process");
+                session->close("Error creating child process for login shell.");
             }
             return;
         }
         if (pid == 0)
         {
             // CHILD
 
-            // sets the effective user ID
-            // as service user
-            bool isUidSet = false;
             if (auto userName = session->getUserName(); !userName.empty())
             {
-                if (struct passwd* pw = getpwnam(userName.c_str());
-                    pw != nullptr)
-                {
-                    int uidr = setuid(pw->pw_uid);
-                    if (!uidr)
-                    {
-                        isUidSet = true;
-                    }
-                    else
-                    {
-                        BMCWEB_LOG_ERROR << "sets service user failed Error: "
-                                         << uidr;
-                    }
-                }
-                else
-                {
-                    BMCWEB_LOG_ERROR << "getpwnam return null passwd";
-                }
+                execl("/bin/login", "/bin/login", "-f", userName.c_str(), NULL);
+                // execl only returns on fail
+                BMCWEB_LOG_ERROR << "unable to replaces current process, execl "
+                                    "fail, Error number:"
+                                 << errno;
+                session->close("Error unable to replaces current process");
```

Comment:
This error message seems a bit uninformative. If you found this in a log file and you weren't familiar with what generated it, how would you respond?